src/main/python/hydra/lib/hydrabase.py

```diff
@@ -666,6 +671,66 @@ def get_mesos_slave_ips_attr(self, attr_type, attr_value):
         """
         return self.__mesos.get_slave_ips_from_attribute(attr_type, attr_value)
 
+    def get_all_mesos_slave_attr(self):
```

Comment:
So, we have a dictionary datastructure named 'info'. We are putting info dict values to another list named attr_list. And then we are using that list to do whatever we want to do.
The thing which I was trying to ask is that can we avoid the intermediate list attr_list DS? Can't we directly use the info dict.